README.md

```diff
@@ -1,3 +1,26 @@
 # Uninstall Spotify for Windows
 
-Just download and run [Uninstall-Spotify.bat](https://raw.githack.com/amd64fox/Uninstall-Spotify/main/Uninstall-Spotify.bat)
+If you're looking to uninstall Spotify from your Windows system, you can use the provided batch script.
+
+## Usage
+
+1. Download the batch script: [Uninstall-Spotify.bat](https://raw.githack.com/amd64fox/Uninstall-Spotify/main/Uninstall-Spotify.bat).
+2. Double-click the downloaded `Uninstall-Spotify.bat` file to run it.
+3. Follow any on-screen prompts or instructions that may appear.
```

Comment:
delete this line
upon execution, the script does not provide on-screen prompts or instructions;